test/factories/questionaries.rb

```diff
@@ -12,9 +12,12 @@
     avatar { generate :file }
     experience { generate :string }
     want_to_do { generate :string }
-    state 'unviewed'
+    state { Questionary.state_machines[:state].states.map(&:name).first.to_s }
+    member_state { Questionary.state_machines[:member_state].states.map(&:name).first.to_s }
     password { generate :string }
+    role { Questionary.role.values.first }
     password_confirmation { password }
+    type 'Questionary'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.